examples/todo/napi/main.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+import * as slint from "slint-ui";
+
+// import * as demo from "../ui/todo.slint";
+
+let app = slint.create("../ui/todo.slint");
```

Comment:
For now I think we indeed have to settle for this kind of explicit API, and I think that's fine. The old `require("foo.slint")` was a bit of a hack. Perhaps in the future we can use import attributes API (`import("foo.slint", { with: { type: "slint" } })`), but it's too early for that.